Packs/Pulsedive/Integrations/Pulsedive/Pulsedive.py

```diff
@@ -431,9 +431,9 @@ def scan_result_command(client: Client, args: Dict[str, Any], api_key) -> List[C
 
                     ip_indicator = Common.IP(
                         ip=qid_data['data']['indicator'],
-                        asn=qid_data['data']['properties']['geo']['asn'],
-                        geo_country=qid_data['data']['properties']['geo']['country'],
-                        port=qid_data['data']['attributes']['port'],
+                        asn=qid_data['data']['properties']['geo'].get('asn'),
```

Comment:
```suggestion
                        asn=qid_data.get('data', {}).get('properties', {}).get('geo', {}).get('asn'),
```
What I meant is, that let's change all the mentioned lines (434-436, 461-463) to access the dict keys by the 'get' method i.e `dict.get(key, {})` rather than `dict[key]`. 
I think it'll be much safer to access those fields as we're trying to access deep into the dict and we should be able to handle the case where the dict level is shalower.